src/java.base/share/classes/java/security/doc-files/debug-system-property.html

```diff
@@ -52,24 +52,10 @@ <h1><B>{@systemProperty java.security.debug}</B></h1>
 <p> To monitor security access, you can set the <code>java.security.debug</code>
     system property, which determines what trace messages are printed during
     execution. The value of the property is one or more options separated by a
-    comma.
+    comma. Each trace message includes the thread id, timestamp and caller
+    information.
```

Comment:
Small comment, but I would list this in the order they are printed: "thread id, caller information, and timestamp".